packages/utils/src/requestdata.ts

```diff
@@ -295,13 +295,13 @@ export function addRequestDataToEvent(
 ): Event {
   const include = {
     ...DEFAULT_INCLUDES,
-    ...options?.include,
+    ...(options ? options.include : undefined),
```

Comment:
l: I think this can be shortened even further:
```suggestion
    ...(options && options.include),
```